plugin/validator_test.go

```diff
@@ -1913,15 +1913,28 @@ func TestValidator_ValidateAllTypedefs(t *testing.T) {
 			expectedErr: `{"config":{"semantic_version":"must not have empty version segments"}}`,
 		},
 		{
-			name: "lua code not accepted",
+			name: "valid lua code gets accepted",
 			config: `{
 				"lua_code": {"header": "return nil"}
 			}`,
+		},
+		{
+			name: "invalid lua code gets rejected",
+			config: `{
+				"lua_code": "test"
+			}`,
+			wantErr:     true,
+			expectedErr: `{"config":{"lua_code":"expected a map"}}`,
+		},
+		{
+			name: "invalid lua code syntax gets rejected",
+			config: `{
+				"lua_code": {"header": "os.execute('echo 10)"}
+			}`,
```

Comment:
Please add a test for 'valid lua code but invalid based on sandboxing rules gets rejected'.

Reply:
done